src/lib.rs

```diff
@@ -623,14 +623,39 @@ impl Image {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct TilesContainer {
+    pub data: Vec<u32>,
+    pub tiles_per_row: u32,
+    pub number_of_rows: u32,
+}
+
+impl From<&TilesContainer> for Vec<Vec<u32>> {
```

Comment:
I don't think we need this converter. What use would people have for converting to this sub-optimal data structure? Maybe instead it could be useful to provide a getter to retrieve a tile given `x` and `y` coordinates?